app/controllers/application_controller.rb

```diff
@@ -73,11 +73,9 @@ def current_user
     super || guest_user
   end
 
-  #find the guest user account if a guest user session is currently active
+  # find the guest user account if a guest user session is currently active
   def guest_user
-    unless session[:guest_user_id].nil?
-      User.where(id: session[:guest_user_id]).first
-    end
+    User.find_by(id: session[:guest_user_id])
```

Comment:
Exact same query in SQL, just shorter way of writing it. Will give nil if not present, so same function.

Reply:
Nice.  Would `.where(id: session[:guest_user_id]).first` do the same?

Reply:
Yes.